content/en/blog/_posts/2020-09-04-introducing-structured-logs.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog 
+title: 'Introducing Structured Logs'
+date: 2020-09-04 
+slug: kubernetes-1-19-Introducing-Structured-Logs
+---
+
+**Authors:** Marek Siarkowicz (Google), Nathan Beach (Google)
+
+Logs are an essential aspect of observability and a critical tool for debugging. But Kubernetes logs have traditionally been unstructured strings, making any automated parsing difficult and any downstream processing, analysis, or querying challenging to do reliably.
+
+In Kubernetes 1.19, we are adding support for structured logs, which natively support (key, value) pairs and object references. We have also updated many logging calls such that over 99% of logging volume in a typical deployment are now migrated to the structured format.
```

Comment:
In Kubernetes 1.19, we are adding support for [structured logs](/docs/concepts/cluster-administration/system-logs/), which natively support (key, value) pairs and object references.